diff --git a/dist/cjs/plugins/deploy/platforms/vercel.js b/dist/cjs/plugins/deploy/platforms/vercel.js
index f3967a12fd9f84acd67a568f069c7a2e3918f404..34d4aa17e1eed48a7b50390debcebe36cf5d1ccf 100644
--- a/dist/cjs/plugins/deploy/platforms/vercel.js
+++ b/dist/cjs/plugins/deploy/platforms/vercel.js
@@ -106,7 +106,7 @@ const createVercelPreset = (appContext, modernConfig, needModernServer) => {
         });
         const nodeVersion = process.versions.node.split(".")[0];
         await import_utils.fs.writeJSON(import_node_path.default.join(funcsDirectory, ".vc-config.json"), {
-          runtime: nodeVersion,
+          runtime: `nodejs${nodeVersion}.x`,
           handler: "index.js",
           launcherType: "Nodejs",
           shouldAddHelpers: false,
diff --git a/dist/esm/plugins/deploy/platforms/vercel.js b/dist/esm/plugins/deploy/platforms/vercel.js
index 697dc85095d94581b89b5fd82291cdbb3ff4af15..cf2410f91b4d0677c48c1bb427a4b5be6e277552 100644
--- a/dist/esm/plugins/deploy/platforms/vercel.js
+++ b/dist/esm/plugins/deploy/platforms/vercel.js
@@ -137,7 +137,7 @@ var createVercelPreset = function(appContext, modernConfig, needModernServer) {
               return [
                 4,
                 fse.writeJSON(path.join(funcsDirectory, ".vc-config.json"), {
-                  runtime: nodeVersion,
+                  runtime: "nodejs".concat(nodeVersion, ".x"),
                   handler: "index.js",
                   launcherType: "Nodejs",
                   shouldAddHelpers: false,
diff --git a/dist/esm-node/plugins/deploy/platforms/vercel.js b/dist/esm-node/plugins/deploy/platforms/vercel.js
index 50081a63b2516c9148747f14d4b40337f054785b..299309db19a98c797c6acd77c2160ca61309f9dc 100644
--- a/dist/esm-node/plugins/deploy/platforms/vercel.js
+++ b/dist/esm-node/plugins/deploy/platforms/vercel.js
@@ -73,7 +73,7 @@ const createVercelPreset = (appContext, modernConfig, needModernServer) => {
         });
         const nodeVersion = process.versions.node.split(".")[0];
         await fse.writeJSON(path.join(funcsDirectory, ".vc-config.json"), {
-          runtime: nodeVersion,
+          runtime: `nodejs${nodeVersion}.x`,
           handler: "index.js",
           launcherType: "Nodejs",
           shouldAddHelpers: false,
